exampleSite/config/_default/params.toml

```diff
@@ -30,6 +30,8 @@ listDateFormat = "2/1/2006"
 
 reversepagetitle = true # When set to 'true', the Window Title will be reversed to 'Title | Author' instead of the default 'Author | Title'
 
+hidesidebar = true
```

Comment:
How about letting the user decide by providing a toggle in the menu?

Reply:
@lxndrblz 
That's fine. I'm just thinking that, when deciding for the user toggle, you don't have to overthink the options for the site owner.